marketplace/deployer_util/config_helper.py

```diff
@@ -119,7 +124,12 @@ def __init__(self, dictionary):
 
   def validate(self):
     """Fully validates the schema, raising InvalidSchema if fails."""
-    if self.app_api_version is None:
+    is_v2 = False
+    if self._x_google_marketplace is not None:
+      self._x_google_marketplace.validate()
```

Comment:
would this function raise an exception or something? feels weird to have this function call alone with no return statement, no parameters, no try catch... seems like we don't know what to expect from it.
Maybe include validate in the is_v2 function code block?